app/views/external_users/claims/offence_details/_fee_reform_fields.html.haml

```diff
@@ -16,22 +16,19 @@
         .form-group
           = govuk_button(t('buttons.continue'), name: 'commit_continue')
 
-          = govuk_link_button_secondary('#', class: 'fx-clear-selection') do
-            = t('clear_selection', scope: "#{locale_scope}.actions")
+          = govuk_link_button_secondary(t('clear_selection', scope: "#{locale_scope}.actions"), '#', class: 'fx-clear-selection')
 
-.form-group.mod-search-input
-  %label.form-label{ for: 'offence' }
-    = 'Search for the offence'
-    %span.form-hint
-      = 'For example class, band, offence or act name'
+= f.govuk_text_field :offence,
+  value: nil,
+  class: 'fx-input',
+  form_group: { classes: 'mod-search-input' },
+  hint: { text: 'For example class, band, offence or act name' },
+  label: { text: 'Search for the offence' }
```

Comment:
Minor: Could we move these text values to the relevant locale file

Reply:
Done! 👍🏼